server/models/User.js

```diff
@@ -351,7 +351,8 @@ export default (Sequelize, DataTypes) => {
 
         let update = false;
         const allowedFields = 
-          [ 'firstName',
+          [ 'username',
```

Comment:
Don't know ¯\_(ツ)_/¯ 
Weren't you checking that when saving the user? Where did you put that logic?

If there is an error message that bubbles up it will be shown on the frontend.